mail_tracking_mass_mailing/README.rst

```diff
@@ -20,6 +19,21 @@ This addon will be automatically installed when 'mail_tracking' and
 Usage
 =====
 
+From mail statistic object, you'll can see:
+- Email tracking state
+- Email related tracking object
+- Email related tracking events
+
+From mass mailing contact, you'll can see:
+- Email score, in order to clean up your lists from bad score emails
+
+As a bonus feature, you have a new checkbox 'Avoid resend' in mass mailing,
+in order to not send twice the same email to the same recipient. This is very
+useful when you want to resend the mass mailing after changing selection
+recipients. Notice that recipient selection could be a domain over a model, so
+result ids could change over the time. With this flag you can send
+the same email several times but only once to each recipient.
```

Comment:
But this is only effective if the mass mailing is put again as not send, and you click on "Send all" again. You should specify it.